meetings/2024/2024-04-30.md

```diff
@@ -0,0 +1,110 @@
+# OpenJS Foundation Cross Project Council Meeting 2024-04-30
+
+## Links
+
+* **Recording**:
+* **GitHub Issue**: https://github.com/openjs-foundation/cross-project-council/issues/1301
+
+## Present
+
+* OpenJS Foundation Cross Project Council
+* OpenJS Foundation Project Maintainers
+* OpenJS Foundation Board of Directors
+
+
+* Ulises Gascon (@UlisesGascon)
+* Jordan Harband (@ljharb)
+* Tobie Langel (@tobie)
+* Claudio Wunder (@ovflowd)
+* Adrian Estrada (@edsadr)
+* Paula Paul (@paulapaul)
+* Mike Samuel (@mikesamuel)
+* Chris de Almeida (@ctcpip)
+* Ben Sternthal (@bensternthal)
+* Joey Guerra (@joeyguerra)
+* Michael Dawson (@mhdawson)
+* Marco Ippolito (@marco-ippolito)
+* Rich Trott (@Trott)
+
+## Agenda
+
+### Announcements
+
+* Standard meeting following this meeting with special guest! - Chris
+Mikhail from University of Bergen to talk about TC39-TG5 - Experiments in Programming Language Standardization
+* Travel fund update - Ben
+* Congrats to re-elected CPC Board members!
+
+### Board Meeting Updates
+
+- https://github.com/openjs-foundation/cross-project-council/labels/waiting-on-board
+- Board votes will occur in two weeks (from prior board meeting) on items sent to board for approval
+- Note, there is discussion on ‘in kind’ membership, Ben and Robin to come forward with recommendations
+
+### Staff Updates
+
+- https://github.com/openjs-foundation/cross-project-council/labels/waiting-on-legal-info
+
+https://github.com/openjs-foundation/cross-project-council/issues/1133 was tagged to CPC, and noted it may be related to Ecosystem and Sunsetting discussions
+
+- https://github.com/openjs-foundation/cross-project-council/labels/waiting-on-staff-update
+- https://github.com/openjs-foundation/cross-project-council/labels/waiting-on-website-update
+
+*Extracted from **cross-project-council-agenda** labeled issues and pull requests from the **openjs-foundation org** prior to the meeting.
```

Comment:
Should we change `*Extracted` to `* Extracted` in whatever template is being used since this seems to appear in every meeting minutes documents PRs? Or is the current writing intended?

Additionally should we make `**cross-project-council-agenda**` point to the GitHub label like https://github.com/openjs-foundation/cross-project-council/labels/cross-project-council-agenda?

Reply:
Good idea! Filed here: https://github.com/nodejs/create-node-meeting-artifacts/pull/160